File: truncate-sentence/truncate-sentence.py
class Solution:
    def truncateSentence(self, s: str, k: int) -> str:
        
        first_array_of_words = s.split(" ")
        final_array_of_words = first_array_of_words[:k]
        
        return " ".join(final_array_of_words)

File: truncate-sentence/truncate-sentence.js
/**
 * @param {string} s
 * @param {number} k
 * @return {string}
 */
var truncateSentence = function(s, k) {
    let first_array = s.split(" ");
    let final_array = first_array.slice(0, k);
    return final_array.join(" ");
};
